x/ethbridge/keeper/blacklist_test.go

```diff
@@ -13,8 +16,9 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// Setting to true to use 0.45
 var (
-	isFeatureToggleSdk045 = os.Getenv("FEATURE_TOGGLE_SDK_045") != ""
+	isFeatureToggleSdk045 = os.Getenv("FEATURE_TOGGLE_SDK_045") == ""
```

Comment:
Can we get rid of this toggle now?

Reply:
correct we should get rid of this toggle as well